subsys/bluetooth/host/gatt.c

```diff
@@ -1892,6 +1892,52 @@ static u8_t disconnected_cb(const struct bt_gatt_attr *attr, void *user_data)
 	return BT_GATT_ITER_CONTINUE;
 }
 
+bool bt_gatt_subscribed(struct bt_conn *conn,
```

Comment:
I generally prefer as short (but still descriptive) names as possible, but we usually try to have a verb in there, which in this case would be "is", so would you consider `bt_gatt_is_subscribed()` instead?

Reply:
I was considering it. So yes, I can make it _is_.